### PR TITLE
Merge OSL shader support.

### PR DESCRIPTION
**Pull request by MWY3510**
Let me know if you have any questions. Sorry this is late.

### CODE REVIEW

File: nodes.py [hunk 1]
```diff
@@ -24,18 +24,25 @@
 # ##### END MIT LICENSE BLOCK #####
 
 import bpy
+import _cycles
```
Comment (bsavery) [inline nodes.py]: This is just funny...

Comment (MWY3510) [inline nodes.py]: Why is that?

Comment (bsavery) [inline nodes.py]: Funny, hilarious to use cycles in a PRMan addon.  Made me laugh anyway...

**Comment by bsavery** [comment]
Ok only issue I see is a few prints in the gen_params method...

**Comment by MWY3510** [comment]
Sorry about that I forgot about the prints.

**Comment by bsavery** [comment]
Issue working with IPR might just be the shader path???

File: nodes.py (hunk 1 above)
Comment (MWY3510) [inline nodes.py]: Now that you mention it, this is funny.

**Comment by bsavery** [comment]
Think I fixed IPR.  Just a paths problem.  Note that when IPR is started, it's not CWD'ing so need full paths.... I should fix that.

**Pull request merged**